advanced/am-pattern-2/README.md

```diff
@@ -39,16 +43,19 @@
 ### Install Chart From [WSO2 Helm Chart Repository](https://hub.helm.sh/charts/wso2)
 
 ##### 1. Deploy Helm chart for WSO2 APIM Pattern 2 deployment
+
 [Option 1] Deploy using Docker images from DockerHub.
```

Comment:
these options can we tab it and give a different number for all the places you have [option]

Reply:
We can make this issue as Resolved